examples/panorama.html

```diff
@@ -75,13 +75,14 @@
                         protocol: 'static',
                         id: _id,
                         projection: 'EPSG:4326',
+                    }).then(() => {
+                        view.notifyChange(false);
```

Comment:
I'm wondering if this shouldn't be done automatically in `addLayer` (or with `whenReady` in _preprocessDataLayer`)?

Reply:
Indeed it could be done in `whenReady`. Do you want me to do this in this PR?

Reply:
We can leave it like it is here, and remove it in another PR. Whatever suits you best :-)